src/Tracker/utils/index.js

```diff
@@ -0,0 +1,45 @@
+export const sendAnalytics = (payload) => {
+  const request = new XMLHttpRequest()
+  request.open('POST', 'https://analytics-sdk-dev.onfido.com/v1/import')
+  request.setRequestHeader('Content-Type', "application/json")
+  request.setRequestHeader('Authorization', "Basic M002Z3tQWURwWTlUanlDKDNRTHIqTDZOQ1hVY2FLZUU6")
+
+  request.onload = () => {
+    if (request.status === 200 || request.status === 201) {
+      console.log(request.response)
+    }
+    else {
+      console.log(request)
+    }
+  }
+  request.onerror = () => console.log(request)
+
+  request.send(payload)
+}
+
+export const formatAnalytics = (woopra, eventName, properties) =>
+  JSON.stringify({
+    batch: [{
+      anonymousId: woopra.cookie,
+      channel: "Web SDK",
+      context: {
+        app: {
+          name: woopra.instanceName,
+          namespace: woopra.options.domain
+        },
+        library: {
+          name: "analytics-js", //made up values, I don't know what this is
+          version: "0.0.0" //made up values, I don't know what this is
```

Comment:
Potentially these 2 properties should be our SDK's name and version?

Reply:
I think this refers to the library we used to capture the events, so in this case it would be woopra. I'm not sure if we care about capturing these values. The sdk version is captured inside `woopra. visitorData`